src/test/java/com/oya/kr/test/mapper/TimeMapperTest.java

```diff
@@ -0,0 +1,35 @@
+package com.oya.kr.test.mapper;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+
+import com.oya.kr.global.config.RootConfig;
+
+import lombok.extern.log4j.Log4j;
+
+@WebAppConfiguration
+@RunWith(SpringRunner.class)
+@ContextConfiguration(classes = {RootConfig.class})
+@Log4j
+public class TimeMapperTest {
+
+	@Autowired
+	private TimeMapper timeMapper;
+
+	@Test
+	public void testGetTime() {
+		log.info(timeMapper.getClass().getName());
+		log.info(timeMapper.getTime());
+	}
```

Comment:
```suggestion
	@DisplayName("테스트 이름")
	@Test
	public void getTime() {
		log.info(timeMapper.getClass().getName());
		log.info(timeMapper.getTime());
	}
```
jUnit5 부터는 테스트 메서드명에 `test` 를 붙이지 않아도 됩니당 그리고 테스트 시에는 log 가 아닌 assertj 등 실제 테스트 코드로 작성해주세요!